workshops/templates/workshops/airport.html

```diff
@@ -11,7 +11,15 @@
 </table>
 
 <div class="clearfix">
+  {% if perms.workshops.change_airport %}
   <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary">Edit</a></p>
+  {% else %}
+  <p class="edit-object pull-left"><a href="{% url 'airport_edit' airport.iata %}" class=" btn btn-primary disabled">Edit</a></p>
+  {% endif %}
```

Comment:
What about this instead:

~~~
<p class="..."><a href="{% url ... %}" class="btn btn-primary {%if not perms.workshops.change_airport %}disabled{% endif %}">Edit</a></p>
~~~

That cuts it from 5 lines to 1 and makes the intention clearer by removing a lot of duplication.  (I had to scan the line a couple of times to convince myself there was only one significant difference.)